vcl_templates/assets.vcl.erb

```diff
@@ -153,6 +153,11 @@ sub vcl_recv {
      error 801 "Force SSL";
   }
 
+  # Serve and log an empty response for analytics purposes
+  if (req.url ~ "^/static/a") {
```

Comment:
Won't this capture any url beginning with 'static/a' as it's unterminated? Is it possible to scope this more tightly?